tests/test_aea.py

```diff
@@ -436,8 +436,6 @@ def setup_class(cls):
         resources = Resources()
         resources.add_component(Skill.from_dir(Path(CUR_PATH, "data", "dummy_skill")))
         identity = Identity(agent_name, address=wallet.addresses[FETCHAI])
-        cls.input_file = tempfile.mkstemp()[1]
```

Comment:
Why are they removed?

Reply:
because they were not needed (we use OEFLocalConnection)